pom.xml

```diff
@@ -2,9 +2,9 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.atlassian.sourcemap</groupId>
+    <groupId>com.evergage.thirdparty</groupId>
```

Comment:
change to `com.evergage.thirdparty.atlassian`

Reply:
@ispringer should this be `atlassian` or `sourcemap`?

Reply:
I like `atlassian`, but you could go either way.

Reply:
@ispringer I went with `sourcemap` because I think that matches Oleg's concerns about namespace scoping.